### ISSUE TITLE
Changing directions is awkward

### ISSUE THREAD

**Issue by dantogno** [parkerhams/SystemsStudyGame#8]
While the player holds the move button, if they try to change directions, the player takes too long to stop moving in the old direction and begin moving in the new one.

**Comment by dantogno** [comment]
I think this can be fixed by:
- [ ] remove the `rotationDrag` on PlayerMovement line 19, and all the other spots you're using that. I think we need to just add some amount of drag to the player rigidbody that's always there by changing the value in the editor. Maybe 2 or something, play with it.
- [ ] Adjust the `movingDrag` in the editor for the player. It needs to be above zero, or we will always go forever in the old direction. Drag is what slows us down, so we need some. You could try 2 or 3 or something. Play with it!
- [ ] Adjust the `thrust` in the editor for the player. You will probably need to bump it up to overcome the drag we are adding. Now that we have max speed limit, you can make add more here without stuff getting crazy. 

In summary, we want faster thrust / acceleration, and more drag.
